Makefile.am

```diff
@@ -12,43 +12,43 @@ bin/hoot:
 bin/hoot.bin:
 
 osmapidb:
-	$(MAKE) -f Makefile.old osmapidb
+	$(MAKE) -f Makefile.hoot osmapidb
```

Comment:
Just `Makefile.old` to `Makefile.hoot` because `old` probably only means pre-automake tools and is the worst name ever.